indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Exception.kt

```diff
@@ -11,7 +11,7 @@ open class Asn1Exception(message: String?, cause: Throwable?) : Throwable(messag
 class Asn1TagMismatchException(val expected: Asn1Element.Tag, val actual: Asn1Element.Tag, detailedMessage: String? = null) :
     Asn1Exception((detailedMessage?.let { "$it " } ?: "") + "Expected tag $expected, is: $actual")
 
-class Asn1StructuralException(message: String) : Asn1Exception(message)
+class Asn1StructuralException(message: String, override val cause: Throwable? = null) : Asn1Exception(message, cause)
```

Comment:
why is this `override val` if it is passed into the underlying constructor anyway

Reply:
IDE assistant mishap